src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -102,7 +102,6 @@ private static void ComputeDeclarations(
                 case SyntaxKind.StructDeclaration:
                 case SyntaxKind.RecordDeclaration:
                 case SyntaxKind.RecordStructDeclaration:
-                    // PROTOTYPE likely needs work for analyzers
```

Comment:
I imagined we would keep the comments, just replace "PROTOTYPE" with a link to tracking issue.

Reply:
That's not the plan I had in mind, but that works. Let me take another stab

Reply:
Done. The search&replace was easier than manually deleting the comments :-)